monkey/monkey_island/cc/services/attack/technique_reports/T1154.py

```diff
@@ -0,0 +1,37 @@
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+from monkey_island.cc.database import mongo
+from common.utils.attack_utils import ScanStatus
+from common.data.post_breach_consts import POST_BREACH_TRAP_COMMAND
+
+
+__author__ = "shreyamalviya"
+
+
+class T1154(AttackTechnique):
+    tech_id = "T1154"
+    unscanned_msg = "Monkey did not use the trap command on the system."
+    scanned_msg = "Monkey tried using the trap command but failed on the system."
+    used_msg = "Monkey used the trap command on the system."
+
+    query = [{'$match': {'telem_category': 'post_breach',
+                         'data.name': POST_BREACH_TRAP_COMMAND}},
+             {'$project': {'_id': 0,
+                           'machine': {'hostname': '$data.hostname',
+                                       'ips': ['$data.ip']},
+                           'result': '$data.result'}}]
```

Comment:
I wonder how many queries like this we have in our code. Maybe we should create an attack technique query generator class